src/components/atoms/TooltipMarker.tsx

```diff
@@ -116,11 +115,10 @@ const useStyles = makeStyles({
     transform: (prop) => getLabelArrowRotation(prop.offset as ClockPosition),
   },
 });
-const TooltipMarker = ({ data, position, offset }: any) => {
-  const { i18n } = useTranslation();
-  const order = i18n.language === 'en'
-  const classes = useStyles({offset, order});
-  const iconText: any = `${dateFormat(data.accident_timestamp)}`;
+const TooltipMarker = ({ data, position, offset, locale}: any) => {
+  const classes = useStyles(offset);
```

Comment:
if we don't use `order` as a props to the style, let's remove it from:
Line 103 - `order: (prop: any) => (prop.order ? 0 : 1),` 

also it's not an object with `offset` anymore here:
Line 96 - `transform: (prop) => getLabelPosition(prop.offset as ClockPosition),`
Line 98, 115

it's `offset` instead of `prop`, but better wrap `useStyles(offset);` as an object like that 
`useStyles({ offset });`